### ISSUE TITLE
Xcode7.2 swift before 2.2

### ISSUE THREAD

**Issue by tommybiteme** [DanTheMan827/ios-app-signer#31]
My Xcode version is 7.2 so i find a problem when i run it.The problem is from MainView.swift where #selector();
Before swift 2.2 the usage of selector is Selector().
Replace it and get it.
Good luck!

**Comment by DanTheMan827** [comment]
Yeah, I haven't touched the code since before the new swift versions came out...

I just have to run the upgrade utility in xcode
